infra/dshop-control-client/src/components/Edit/DeployWizard/Password.js

```diff
@@ -1,6 +1,6 @@
 import React, { useEffect, useState } from 'react'
 import { useStoreState } from 'pullstate'
-import axios from 'axios'
+import axios from 'utils/axiosWithCredentials'
```

Comment:
There was a reason I separated these, but can't remember exactly why. Does the flow still work if they've never registered?

Reply:
It was actively breaking things without that setting.  And it shouldn't cause any issues.  I believe it just means cookies and certain headers can be used, which you need when you're logging in to things or depending on cookies for auth.